### PR TITLE
Fix for issue 5: Support Gradle 7

### PR DESCRIPTION
**Pull request by saruye**
I added the annotations with were missing to work with newer gradle versions. I didn't look too much if these are the best annotations. It also seems like it is a bit redundant to have the config and the single properties in the tasks. 
But for me the solution is working.

### CODE REVIEW

File: i18n4k-gradle-plugin/src/main/kotlin/de/comahe/i18n4k/gradle/plugin/GenerateI18n4kFilesTask.kt [hunk 1]
```diff
@@ -43,7 +41,7 @@ open class GenerateI18n4kFilesTask : DefaultTask() {
 
     @InputDirectory
     open fun getInputDirectory(): File {
-        var path = config.inputDirectory;
```
Comment (saruye) [inline i18n4k-gradle-plugin/src/main/kotlin/de/comahe/i18n4k/gradle/plugin/GenerateI18n4kFilesTask.kt:46]: Fixed a warning from IntelliJ here

File: i18n4k-gradle-plugin/src/main/kotlin/de/comahe/i18n4k/gradle/plugin/GenerateI18n4kFilesTask.kt [hunk 2]
```diff
@@ -5,17 +5,15 @@ import de.comahe.i18n4k.generator.GenerationTargetPlatform
...
+import org.gradle.api.tasks.*
 import java.io.File
 
 
 @Suppress("unused")
 open class GenerateI18n4kFilesTask : DefaultTask() {
 
 
+    @Nested
     lateinit var config: I18n4kExtension
```
Comment (comahe-de) [inline i18n4k-gradle-plugin/src/main/kotlin/de/comahe/i18n4k/gradle/plugin/GenerateI18n4kFilesTask.kt:17]: Hi @saruye,
Thanks for your contribution. But an `@Internal` annotation at the `config` member would be the better solution, as `config` is used for the other parameters of the task.

**Comment by saruye** [comment]
@comahe-de thanks for the feedback. Just updated this quickly. Didn't test it though. If I should spend some more time, I can try to do it later today.

**Comment by comahe-de** [comment]
@saruye The changes in `I18n4kExtension.kt` can be reverted with the change on the `config` member. It would be good if you could test it, if it is still working with without warning in Gradle 7, because I'm pretty busy in the next weeks.

**Comment by comahe-de** [comment]
Fixed in commit 759dd4736af6d3188db20f67fd122e09894c1766

**Pull request closed without merging**